### DISCUSSION TITLE
Deprecating Slack?

### DISCUSSION THREAD

**Discussion by ploum** [regolith-linux/regolith-desktop#899] [General]
I just realized a message from @kgilmer saying that Slack was deprecated. Could you confirm that here is the place to replace it and that I can permanently remove my slack account? (Regolith being the last Slack I have ;-) )

I’m a bit confused because the activity on the Slack is very low and I missed the announcement of the Slack deprecation. Maybe the Slack itself could be "frozen" to avoid people asking in an empty room? (I don’t know how Slack handle that)

I want to add that Slack is still mentioned in the bug template report. Should this be considered as a bug?

**Comment by cfsmp3** [comment]
Slack is where we hang out AFAIK, I don't think we can deprecate it without a replacement?

**Comment by ploum** [comment]
On 23/09/06 09:59, Carlos Fernandez Sanz wrote:
>Slack is where we hang out AFAIK, I don't think we can deprecate it 
>without a replacement?

I thought that Ken told that this place is the slack replacement. Did I 
misunderstood?

**Reply by kgilmer** [reply]
There is a lot of infra changes I'd like to make but they're always on the back burner.  IMO "[enshitifcation](https://en.wiktionary.org/wiki/enshittification)" is real, and the risks and costs need to be planned for.  I'd like to move to a setup where all software services the Regolith project relies on are self-hostable and under the control of the project.  Github being the big one, but also slack and other things.  I don't believe the slack instance is managed (but could be wrong!) and I don't think new people can (easily) get there.  GH Discussions were setup to see if that would be a more accessible way of facilitating Q&A and other stuff.  It seems to be mostly working.  I would like to move away from slack to some other interactive chat thing but don't know which one is the best fit for the project.  So, given the indecision and lack of time, the status quo remains.

**Comment by ploum** [comment]
On 23/09/06 09:01, Ken Gilmer wrote:
>There is a lot of infra changes I'd like to make but they're always on 
>the back burner.  IMO 
>"[enshitifcation](https://en.wiktionary.org/wiki/enshittification)" is 
>real, and the risks and costs need to be planned for.  I'd like to move 
>to a setup where all software services the Regolith project relies on 
>are self-hostable and under the control of the project.  Github being 
>the big one, but also slack and other things.  I don't believe the 
>slack instance is managed (but could be wrong!) and I don't think new 
>people can (easily) get there.  GH Discussions were setup to see if 
>that would be a more accessible way of facilitating Q&A and other 
>stuff.  It seems to be mostly working.  I would like to move away from 
>slack to some other interactive chat thing but don't know which one is 
>the best fit for the project.  So, given the indecision and lack of 
>time, the status quo remains.


Thanks for that great clarification. I do really appreciate your work on 
this (and all the rest) and I know that it is something really hard to 
do.


I’m starting to really appreciate sourcehut and the fact that everything 
can be done by email without creating an account. Discussions happen on 
mailing-list and sourcehut is self-hostable if needed. I also host my 
blog on sourcehut pages (but there are some limitations such as not 
embedding external ressources in your html pages). Chat is done through 
IRC (chat.sr.ht).

While I do really appreciate the "retro-vibe" of sourcehut, I understand 
that this is a turn-off for some people so I don’t know if this would 
fit a project like Regolith.

The most featurefull solution might be gitlab. Gitlab was, at some 
point, integrated with Mattermost. I don’t know if this is still the 
case and if Mattermost still works as Matrix seems to be the new trendy 
solution for decentralized chats.

One important question is also if you think it worths starting from a 
blank page or if everything need to be imported.

Anyway, this is a huge project that needs to be carefully discussed.

**Comment by UnchartedSystems** [comment]
Is there a way to access the old Slack channel? I'm looking for documentation to get Regolith 1.6 working better on Linux Mint 20.3, and without the slack logs I think I'm SOL.

**Comment by ploum** [comment]
On 23/09/08 01:37, UnchartedSystems wrote:
>Is there a way to access the old Slack channel? I'm looking for 
>documentation to get Regolith 1.6 working better on Linux Mint 20.3, 
>and without the slack logs I think I'm SOL.

Link is https://regolith-linux.slack.com/

But messages older than 90 days are lost (this is a free slack channel).

Your message reminds me the importance on having discussions on a free 
and open platform. We all know that data on a propriatary platform will, 
at some point, be lost and are temporary. But, in the heat of the work, 
we forget about it.

**Comment by denny-futurice** [comment]
Is there an invitation link somewhere for this slack space? I'd like to start contributing to Regolith and a shared chat is nice after all.

If replacing Slack is needed, would Mattermost be an option?

**Reply by kgilmer** [reply]
Hi @denny-futurice , looks like the invite link still exists on the [old site](https://regolith-linux.org/), but when I click the link I get an error. 

https://regolith-linux.herokuapp.com/

> If replacing Slack is needed, would Mattermost be an option?

Possibly!  What do you like about it and why do you think it would be good for the regolith project?

**Reply by denny-futurice** [reply]
The herokuapp link i've found before, but it doesn't seem to work anymore.

> Possibly! What do you like about it and why do you think it would be good for the regolith project?

It's mostly a Slack clone^W Work-a-like you can self host, it's fine. TBH personally i'd be just as happy with just a IRC channel and email list, I'm old-fashioned :-)

**Comment by cfsmp3** [comment]
>TBH personally i'd be just as happy with just a IRC channel and email list, I'm old-fashioned :-)

We have a few channels in slack (general, wayland, dev), etc. 

I don't think going back to one channel in IRC is a good solution. Plus IRC won't give you sync between devices, save history (except whatever we save on our logs when we're connected) and so on.

If we replace slack it has to be with something better...
